com.unity.render-pipelines.universal/Editor/ShaderGUI/BaseShaderGUI.cs

```diff
@@ -241,15 +241,30 @@ public override void OnGUI(MaterialEditor materialEditorIn, MaterialProperty[] p
             ShaderPropertiesGUI(material);
         }
 
+        protected virtual uint materialFilter
+        {
+            get
+            {
+                return uint.MaxValue;
+            }
+        }
```

Comment:
```suggestion
        protected virtual uint materialFilter => uint.MaxValue;
```

You can write this is shorter.

Reply:
Fixed at 54b2a5b6d797f3880e573eda16cc30752657637e